app/scripts/directives/metrics.js

```diff
@@ -351,6 +373,9 @@ angular.module('openshiftConsole')
             });
           } else {
             sparklineByMetric[chartId].load(sparklineData);
+            sparklineByMetric[chartId].axis.max({
+              y: yAxisMax
```

Comment:
and i assume when yAxisMax is undefined because the data is bigger that this being undefined will then tell c3 to do its own thing?